### DISCUSSION TITLE
CANT FIND FOUNDRY-FUND-ME

### DISCUSSION THREAD

**Discussion by Maa-ly** [Cyfrin/foundry-full-course-cu#2204] [Q&A] [answered]
hello everyone, i am currently on lesson 7(specifically at Mock Test. i have a problem locating the test file that contains the mock contract. can anyone help by linking it ?

**Comment by Maa-ly** [comment, marked as answer]
well, i just found it. Incase anyone is going the same issue here is the link

https://github.com/Cyfrin/foundry-fund-me-cu/blob/main/test/mock/MockV3Aggregator.sol

**Reply by EngrPips** [reply]
Awesome, I am glad you figured this out.
